packages/wallets/src/components/WalletsCard/WalletsCard.tsx

```diff
@@ -1,15 +1,31 @@
-import React, { useRef } from 'react';
+import React, { useEffect, useRef } from 'react';
 import { THooks } from '../../types';
 import './WalletsCard.scss';
 
 type TProps = {
     isDemo?: THooks.WalletAccountsList['is_virtual'];
+    isOpen?: THooks.WalletAccountsList['is_active'];
     renderHeader: () => React.ReactNode;
 };
 
-const WalletsCard: React.FC<React.PropsWithChildren<TProps>> = ({ children, isDemo = false, renderHeader }) => {
+const WalletsCard: React.FC<React.PropsWithChildren<TProps>> = ({
+    children,
+    isDemo = false,
```

Comment:
Do we need to pass isDemo here ? 
Cant we get it straight ?

Reply:
isDemo which is received from useActiveWalletAccount in `DesktopWalletsList` is passed to both `WalletsCard` and `WalletListCard`. Since it is not only needed in one component wouldn't it be better to call it once from the parent and passed to the respective childrens?

Reply:
If we can call it directly, we should not pass it down. Minimal prop drilling

Reply:
Updated implementation to reduce prop drilling 👍